---
 src/server.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/server.c b/src/server.c
index d842f392..c3e85309 100644
--- a/src/server.c
+++ b/src/server.c
@@ -511,10 +511,6 @@ int main(int argc, char **argv) {
   if (ssl) {
     info.ssl_cert_filepath = cert_path;
     info.ssl_private_key_filepath = key_path;
-    if (strlen(ca_path) > 0) {
-      info.ssl_ca_filepath = ca_path;
-      info.options |= LWS_SERVER_OPTION_REQUIRE_VALID_OPENSSL_CLIENT_CERT;
-    }
 #if LWS_LIBRARY_VERSION_MAJOR >= 2
     info.options |= LWS_SERVER_OPTION_REDIRECT_HTTP_TO_HTTPS;
 #endif
