diff --git a/src/server.c b/src/server.c
index d842f392..e1180318 100644
--- a/src/server.c
+++ b/src/server.c

@@ -487,39 +487,29 @@
   sprintf(server_hdr, "ttyd/%s (libwebsockets/%s)", TTYD_VERSION,
          LWS_LIBRARY_VERSION);
   info.server_string = server_hdr;
 #endif
 #if LWS_LIBRARY_VERSION_NUMBER >= 2001000 && LWS_LIBRARY_VERSION_NUMBER < 4000000
   info.ws_ping_pong_interval = 5;
 #endif
   if (strlen(iface) > 0) {
     info.iface = iface;
     if (endswith(info.iface, ".sock") || endswith(info.iface, ".socket")) {
 #if defined(LWS_USE_UNIX_SOCK) || defined(LWS_WITH_UNIX_SOCK)
      info.options |= LWS_SERVER_OPTION_UNIX_SOCK;
      info.port = 0; // warmcat/libwebsockets#1985
      strncpy(server->socket_path, info.iface, sizeof(server->socket_path) - 1);
 #else
       fprintf(stderr,
               "libwebsockets is not compiled with UNIX domain socket support");
       return -1;
 #endif
    }
 }
 
-#if defined(LWS_OPENSSL_SUPPORT) || defined(LWS_WITH_TLS)
-  if (ssl) {
-    info.ssl_cert_filepath = cert_path;
-    info.ssl_private_key_filepath = key_path;
-    if (strlen(ca_path) > 0) {
-      info.ssl_ca_filepath = ca_path;
-      info.options |= LWS_SERVER_OPTION_REQUIRE_VALID_OPENSSL_CLIENT_CERT;
-    }
 #if LWS_LIBRARY_VERSION_MAJOR >= 2
    info.options |= LWS_SERVER_OPTION_REDIRECT_HTTP_TO_HTTPS;
 #endif
-}
-#endif

  lwsl_notice("ttyd %s (libwebsockets %s)\n", TTYD_VERSION,
              LWS_LIBRARY_VERSION);
