
diff --git a/src/server.c b/src/server.c
index d842f392..6b2a2a0b 100644
--- a/src/server.c
+++ b/src/server.c
@@ -10,7 +10,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <sys/stat.h>
-
+#include <syslog.h>
 #include "utils.h"
 
 #ifndef TTYD_VERSION
@@ -71,7 +71,6 @@ static const struct option options[] = {
     {"ssl", no_argument, NULL, 'S'},
     {"ssl-cert", required_argument, NULL, 'C'},
     {"ssl-key", required_argument, NULL, 'K'},
-    {"ssl-ca", required_argument, NULL, 'A'},
     {"url-arg", no_argument, NULL, 'a'},
     {"readonly", no_argument, NULL, 'R'},
     {"terminal-type", required_argument, NULL, 'T'},
@@ -86,10 +85,10 @@ static const struct option options[] = {
     {NULL, 0, 0, 0}};
 
 #if LWS_LIBRARY_VERSION_NUMBER < 4000000
-static const char *opt_string = "p:i:c:u:g:s:I:b:6aSC:K:A:Rt:T:Om:oBd:vh";
+static const char *opt_string = "p:i:c:u:g:s:I:b:6aSC:K:Rt:T:Om:oBd:vh";
 #endif
 #if LWS_LIBRARY_VERSION_NUMBER >= 4000000
-static const char *opt_string = "p:i:c:u:g:s:I:b:P:6aSC:K:A:Rt:T:Om:oBd:vh";
+static const char *opt_string = "p:i:c:u:g:s:I:b:P:6aSC:K:Rt:T:Om:oBd:vh";
 #endif
 
 static void print_help() {
@@ -126,7 +125,6 @@ static void print_help() {
           "    -S, --ssl               Enable SSL\n"
           "    -C, --ssl-cert          SSL certificate file path\n"
           "    -K, --ssl-key           SSL key file path\n"
-          "    -A, --ssl-ca            SSL CA file path for client certificate verification\n"
 #endif
           "    -d, --debug             Set log level (default: 7)\n"
           "    -v, --version           Print the version and exit\n"
@@ -302,7 +300,6 @@ int main(int argc, char **argv) {
   bool ssl = false;
   char cert_path[1024] = "";
   char key_path[1024] = "";
-  char ca_path[1024] = "";
 
   struct json_object *client_prefs = json_object_new_object();
 
@@ -431,10 +428,6 @@ int main(int argc, char **argv) {
         strncpy(key_path, optarg, sizeof(key_path) - 1);
         key_path[sizeof(key_path) - 1] = '\0';
         break;
-      case 'A':
-        strncpy(ca_path, optarg, sizeof(ca_path) - 1);
-        ca_path[sizeof(ca_path) - 1] = '\0';
-        break;
 #endif
       case 'T':
         strncpy(server->terminal_type, optarg,
@@ -480,7 +473,8 @@ int main(int argc, char **argv) {
     return -1;
   }
 
-  lws_set_log_level(debug_level, NULL);
+  openlog("ttyd", LOG_NDELAY | LOG_PID, LOG_DAEMON);
+  lws_set_log_level(debug_level, lwsl_emit_syslog);
 
 #if LWS_LIBRARY_VERSION_MAJOR >= 2
   char server_hdr[128] = "";
